index.html

```diff
@@ -3,12 +3,48 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.1/css/all.min.css"
+    />
+
     <link rel="stylesheet" href="/styles/style.css" />
```

Comment:
```suggestion
    <link rel="stylesheet" href="./styles/style.css" />
```

Because of path relativity `/` could represent different locations based on the way directories are setup in the computer the code is running on. It means root file so instead add a `.` like so `./styles/style.css` this way, it'll always mean the current directory which is not relative across computers.

What went wrong in the screenshot I sent you was `/` was pointing to a different root file on my laptop and there was no css in that file so it failed.

Read here: https://stackoverflow.com/a/33774151

Reply:
Wow. Thanks. Will effect in subsequent PRs